infrastructures/infrastructure-common/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/util/LinuxInitScriptGenerator.java

```diff
@@ -36,6 +36,8 @@
 
 public class LinuxInitScriptGenerator {
 
+    public static final String DEFAULT_SUFFIX_RM_TO_NODEJAR_URL = ":8080/rest/node.jar";
```

Comment:
I don't think that it is a good idea to hardcode the port. When an admin configures PA with a different port we cannot change unless we change the code.